factcast-site/documentation/content/setup/server/metrics.md

```diff
@@ -108,13 +108,11 @@ At the **time of writing (0.3.10)** the registry events that are counted are:
 
 ### gRPC Metrics
 
-If you're just looking for remote calls and want to see their execution times (including marshalling/demarshalling from protobuf), you can have a look at the metrics automatically added by the [gRPC library](https://yidongnan.github.io/grpc-spring-boot-starter/en/) we use.
+If you're looking for remote calls and their execution times (including marshalling/demarshalling from protobuf), you can have a look at the metrics automatically added by the [gRPC library](https://yidongnan.github.io/grpc-spring-boot-starter/en/).
```

Comment:
this one slipped through. there is noch such thing as THE grpc lib, but it is THE ONE we use. will add a commit later to fix this.